qcore/siteamp_models.py

```diff
@@ -28,9 +28,36 @@
 import numpy as np
 import pandas as pd
 
+from qcore.uncertainties import distributions
+
 ba18_coefs_df = None
 
 
+def amplification_uncertainty(
+    amplification_factors, frequencies, seed=None, std_dev_limit=2
+):
+    """
+    Applies an uncertainty factor to each value in an amplification spectrum
+    :param amplification_factors: numpy array of amplification factors
+    :param frequencies: numpy array of frequencies for the amplification factors
+    :param seed: Seed to use for the prng. None for a random seed
+    :return: amplification factors with uncertainty applied
+    """
+    sigma_x = (
+        0.6167
+        - 0.1495 / (1 + np.exp(-3.6985 * np.log(frequencies / 0.7248)))
+        + 0.3640 / (1 + np.exp(-2.2497 * np.log(frequencies / 13.457)))
+    )
```

Comment:
Is there a paper that could be linked for these numbers? That could be added to the docstring maybe

Reply:
No, these magic numbers are from @sarahjaneneill. She may have a paper to cite though.

Reply:
And here's the magic: https://pubs.geoscienceworld.org/ssa/bssa/article/107/6/2774/519322/Interfrequency-Correlations-among-Fourier-Spectral